sway-lib-std/src/asset_id.sw

```diff
@@ -0,0 +1,17 @@
+//! Helper functions for generating an AssetId
+library;
+
+use ::alias::{AssetId, SubId};
+use ::constants::ZERO_B256;
+use ::contract_id::ContractId;
+use ::hash::sha256;
+
+/// Construct an AssetId from a ContractId and SubId by hashing them together with sha256.
+pub fn construct_asset_id(contract_id: ContractId, sub_id: SubId) -> AssetId {
```

Comment:
This could be used in some of the functions within `token.sw` file but that can be done in a future PR.